### PR TITLE
Review

### PR DESCRIPTION
**Pull request by ddddavis**
Review ready clone of Apple Homepage

### CODE REVIEW

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi @davisdambis     
#### Great Job so far, but there are a few more things required. 
#### General
- [ ] Each image should have its own link. (Not one link for the four images )


#### Implementation details

- [ ] Elements should be aligned with [float](https://css-tricks.com/all-about-floats/) and [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) or [grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/align_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538830/))

### Make changes and submit another code review request.
*Happy coding*

File: style.css [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+  margin: 0;
+  box-sizing: inherit;
+}
+
+a {
+  text-decoration: none;
+}
+
+.main_container {
+  display: block;
```
Comment (kobiyoyo) [inline style.css]: Elements should be aligned with float and flex or grid

File: style.css [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+  z-index: -1;
+}
+
+.main_image_container {
+  height: 538px;
+  background: url(https://web.archive.org/web/20140301004302im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg) 100% 100% no-repeat;
+}
+
+.main_container header {
+  display: block;
```
Comment (kobiyoyo) [inline style.css]: Elements should be aligned with float and flex or grid

File: style.css [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+  font-size: 26px;
+}
+
+.p3:hover {
+  cursor: pointer;
+  text-decoration: underline;
+}
+
+nav {
+  display: block;
```
Comment (kobiyoyo) [inline style.css]: Elements should be aligned with float and flex or grid

**Review by kobiyoyo — CHANGES_REQUESTED** [review]

**Review by fatymahmed — APPROVED** [review]
## Hi @davisdambis,
### Great job making all the requested changes 🎉.

[OPTIONAL]
- For your next projects, avoid general commit messages like _files_ or _review ready_, remember that commit messages should be descriptive [screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_commits.png).

### It's a 👍🏾 from me!
## ✔️ Approved!

Good luck with your next projects.
_Don't forget to submit your project completion form_

**Pull request merged**